application/models/policy_test.go

```diff
@@ -225,6 +226,18 @@ func (ms *ModelSuite) TestPolicy_LoadMembers() {
 	ms.Len(policy.Members, 1)
 }
 
+func idsInOrder(id1, id2 uuid.UUID) string {
+	tmplt := `%s||%s`
```

Comment:
```suggestion
	const template = `%s||%s`
```
I kept trying to read it as "temp less than" and it just didn't make sense!